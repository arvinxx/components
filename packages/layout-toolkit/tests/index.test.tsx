import React from 'react';
import { render } from '@testing-library/react';

import LayoutToolkit from '@arvinxu/layout-toolkit';

describe('LayoutToolkit', () => {
  it('默认状态', () => {
    const { container } = render(<LayoutToolkit />);
    expect(container).toMatchSnapshot();
  });
});
