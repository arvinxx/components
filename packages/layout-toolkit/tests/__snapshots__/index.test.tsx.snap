// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutToolkit 默认状态 1`] = `
<div>
  <div>
    LayoutToolkit
  </div>
</div>
`;
